64_FFT/app_resources/common.hlsl

```diff
@@ -2,7 +2,7 @@
 
 // Unfortunately not every piece of C++14 metaprogramming syntax is available in HLSL 202x
 // https://github.com/microsoft/DirectXShaderCompiler/issues/5751#issuecomment-1800847954
```

Comment:
remove the comment and link, it was about `using` not being available and having to use `typedef` but obviously since then it was resolved